src/internal/scheduled/scheduled.ts

```diff
@@ -25,15 +28,19 @@ export function scheduled<T>(input: ObservableInput<T>, scheduler: SchedulerLike
   if (input != null) {
     if (isInteropObservable(input)) {
       return scheduleObservable(input, scheduler);
-    } else if (isPromise(input)) {
-      return schedulePromise(input, scheduler);
-    } else if (isArrayLike(input)) {
+    }
+    if (isArrayLike(input)) {
       return scheduleArray(input, scheduler);
-    }  else if (isIterable(input) || typeof input === 'string') {
+    }
+    if (isPromise(input)) {
+      return schedulePromise(input, scheduler);
+    }
+    if (isIterable(input)) {
       return scheduleIterable(input, scheduler);
-    } else if (Symbol && Symbol.asyncIterator && typeof (input as any)[Symbol.asyncIterator] === 'function') {
+    }
+    if (isAsyncIterable(input)) {
       return scheduleAsyncIterable(input as any, scheduler);
```

Comment:
```suggestion
      return scheduleAsyncIterable(input, scheduler);
```

Using `isAsyncIterable` as type guard, `input` shouldn't be `AsyncIterable` already?